### PR TITLE
Todoist integration

### PR DESCRIPTION
**Pull request by jeffrhap**

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
 var config = {
     // Lenguage for the mirror (currently not implemented)
-    lenguage : "en",
-    greeting : ["Hi, sexy!"], // An array of greetings to randomly choose from
```
Comment (evancohen) [inline config.js]: Please remove unnecessary config changes from this PR

File: index.html [hunk 2]
```diff
@@ -5,14 +5,17 @@
     <title>Smart Mirror</title>
     <!-- Libraries -->
     <script src="js/annyang.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.10.6/moment.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.11.2/moment.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.11.2/locale/nl.js"></script>
```
Comment (evancohen) [inline index.html]: Not relevant to this PR, please remove the addition of the localization scripts

File: index.html [hunk 3]
```diff
@@ -23,33 +26,35 @@
...
-	<link rel="stylesheet" href="css/weather-icons.css">
+  <link rel="stylesheet" href="css/weather-icons.css">
 
   </head>
   <body ng-controller="MirrorCtrl as commands">
     <div class="top">
       <div class="top-left">
-        <div class="date grey">{{date | date : 'EEEE, MMMM d yyyy'}}</div>
-        <div class="time">{{date | date : 'hh:mm a'}}</div>
+        <div class="date grey">{{date | date : 'EEEE, d MMMM yyyy'}}</div>
```
Comment (evancohen) [inline index.html]: Not relevant to this PR, do not include it.

File: index.html [hunk 4]
```diff
@@ -91,26 +105,16 @@ <h3 ng-show="!(user.name === undefined)">{{user.name}}</h3>
...
-            <dd>Clears out your reminders list.</dd>
-            <dt>Show map</dt>
-            <dd>Shows a map of your current city.</dd>
-            <dt>Show map of ________</dt>
-            <dd>Shows a map of the given location.</dd>
-            <dt>Giphy ________</dt>
-            <dd>Shows a giphy image </dd>
-            <dt>Show xkcd</dt>
-            <dd>Shows the most recent xkcd comic.</dd>
+            <dt>Help</dt>
```
Comment (evancohen) [inline index.html]: Also not relevant to this PR

File: js/annyang.js [hunk 5]
```diff
@@ -136,7 +136,7 @@
       recognition.interimResults = true;
 
       // Sets the language to the default 'en-US'. This can be changed with annyang.setLanguage()
-      recognition.lang = 'en-US';
+      recognition.lang = 'nl-NL';
```
Comment (evancohen) [inline js/annyang.js]: Not relevant to this PR

**Comment by evancohen** [comment]
There are a lot of changes here that are not relevant for this feature implementation. Either fork from the dev branch again and create a new pull request or revert the unnecessary changes.

**Comment by yaydogan** [comment]
@jeffrhap If you want, I can take your code and create this service on a clean fork. I am thinking about some changes:
- planning to display the to do list under calendar
- not planning to add/remove tasks using voice control at the moment
- looking at adding icons for calendar and task entries to differentiate them 

Let me know if you are already working on new PR. Thanks!!!

**Pull request closed without merging**